camera_aravis2/include/camera_aravis2/config_structs.h

```diff
@@ -162,6 +162,22 @@ struct AcquisitionControl
     /// Number of frames to acquire when 'MultiFrame' is selected as acquisition mode.
     int frame_count = 0;
 
+    /// Setting the mode for the selected trigger.
+    std::string trigger_mode = "n/a";
+
+    /// Setting the signal source for the selected trigger.
+    std::string trigger_source = "n/a";
+
+    /// Selects the I/O line to configure. Once a line has been selected, all changes to the line
+    /// settings will be applied to the selected line.
+    std::string line_selector = "n/a";
```

Comment:
i just realised that `LineSelector`, `LineMode` and `LineSource` are under `DigitalIOControl` and not under `AcquisitionControl`.
however, this works just fine since the grouping is completely ignored by aravis when setting the parameters (see #59). i don't have the time to implement this, but i think implementing #57 might be the wiser approach compared to adding yet another struct here for `DigitalIOControl`.
WDYT?